src/sgame/sg_buildable.cpp

```diff
@@ -4153,7 +4153,7 @@ static void BuildableSpawnCBSE(gentity_t *ent, buildable_t buildable) {
 		}
 
 		default:
-			assert(false);
+			ASSERT(false);
```

Comment:
As long as we don't use an enum class here, I think it is good to keep it though few other places in the code do that.